scale/job/seed/manifest.py

```diff
@@ -395,6 +396,15 @@ def validate_connection(self, job_conn):
         if len(self.get_output_files()) and not job_conn.has_workspace():
             raise InvalidConnection('No workspace provided for output files')
         return warnings
+        
+    def validate_workspace_for_outputs(self, exe_config):
+        """Validates the given job's output workspaces
+        :param exe_config: The job configuration
+        
+        :raises :class:`job.data.exceptions.InvalidConnection`: If thre is a configuration problem.
+        """
+        if len(self.get_output_files()) and not exe_config.get_output_workspace_names():
+            raise InvalidConnection('No workspace defined for output files')
```

Comment:
Should be InvalidConfiguration?

Reply:
Changed to job.configuration.data.exceptions.InvalidConfiguration exception.